ZenJourney/app/src/main/java/ca/uwaterloo/cs/MainActivity.kt

```diff
@@ -32,7 +34,7 @@ class MainActivity : ComponentActivity() {
     }
 }
 
-class AppState {
+class AppState(context: Context) {
```

Comment:
nit: if we pass in context we can just store that too and remove it from being passed into some of the pages